diagnostics/include/performance_handler.h

```diff
@@ -42,30 +41,25 @@ enum class Unit {
   COUNT
 };
 
+using namespace boost::histogram;
+using axes_t = std::vector<axis::regular<>>;
+
 // A recorder is a thread-safe type that should always be created in a shared pointer to ensure
 // proper destruction. The recorder is used to add values to the histogram in the recording thread.
 // Interval histograms can be extracted in other threads.
 struct Recorder {
-  // These values, (except for unit), come directly from hdr histogram.
-  // https://github.com/HdrHistogram/HdrHistogram_c/blob/master/src/hdr_interval_recorder.h#L28-L32
-  Recorder(const std::string& name,
-           int64_t lowest_trackable_value,
-           int64_t highest_trackable_value,
-           int significant_figures,
-           Unit unit)
-      : Recorder(lowest_trackable_value, highest_trackable_value, significant_figures, unit) {
+  Recorder(const std::string& name, int64_t highest_trackable_value, Unit unit)
+      : Recorder(highest_trackable_value, unit) {
     this->name = name;
   }
 
-  Recorder(int64_t lowest_trackable_value, int64_t highest_trackable_value, int significant_figures, Unit unit)
-      : unit(unit) {
-    ConcordAssert(lowest_trackable_value > 0);
-    auto rv =
-        hdr_interval_recorder_init_all(&recorder, lowest_trackable_value, highest_trackable_value, significant_figures);
-    ConcordAssertEQ(0, rv);
+  Recorder(int64_t highest_trackable_value, Unit unit) : unit(unit) {
+    recorder = make_histogram(axis::regular<>(highest_trackable_value, 0, highest_trackable_value));
```

Comment:
You will have to revisit this. I believe It's going to degrade performance severely.